src/psyclone/psyad/transformations/preprocess.py

```diff
@@ -45,7 +45,7 @@
 from psyclone.psyir.nodes import (Assignment, IntrinsicCall, Reference)
 from psyclone.psyir.transformations import (DotProduct2CodeTrans,
                                             Matmul2CodeTrans,
-                                            ArrayRange2LoopTrans,
+                                            ArrayAssignment2LoopsTans,
```

Comment:
"...Trans" and again below where you use it.

Reply:
Done